README.md

```diff
@@ -275,8 +275,12 @@ You can specify the browser and Node.js versions by queries (case insensitive):
   `browserslist-config-mycompany` npm package.
 * `supports es6-module`: browsers with support for specific features.
   `es6-module` here is the `feat` parameter at the URL of the [Can I Use]
-  page. A list of all available features can be found at
-  [`caniuse-lite/data/features`].
+  page. This includes browsers with full or partial support for a feature.
+  A list of all available features can be found at [`caniuse-lite/data/features`].
+* `fully supports es6-module`: browsers with support for specific features.
```

Comment:
Can you improve docs to clarify the difference in query description?

Reply:
I can try! 

I already mention that `supports` is for "full or partial support", while "fully supports" says "This only includes browsers with full support for a feature."

I could give an example with something like `supports es6` vs `fully supports es6`.

Did you have any thoughts on what'd be a good way to clarify the difference?

Reply:
> Did you have any thoughts on what'd be a good way to clarify the difference?

I am not native speaker and docs is harder for me 😅